buildbot/osuosl/master/config/builders.py

```diff
@@ -2393,6 +2393,9 @@
                     flang_extra_configure_args=[
                         "-DFLANG_ENABLE_WERROR=ON",
                         "-DCMAKE_BUILD_TYPE=Release",
+                    ],
+                    flang_rt_extra_configure_args=[
+                        "-DCMAKE_BUILD_TYPE=Release",
```

Comment:
The build type needs to be restated for flang_rt because it's being built in a separate step, do I understand correctly?

Reply:
Correct, just like `flang_extra_configure_args`. I could have made getFlangOutOfTreeBuildFactory do it by default, but it didn't to it for the flang step either.